internal/services/bot/bot_channel_directline_resource.go

```diff
@@ -64,12 +64,34 @@ func resourceBotChannelDirectline() *pluginsdk.Resource {
 							ValidateFunc: validation.StringIsNotEmpty,
 						},
 
+						"application_id": {
+							Type:         pluginsdk.TypeString,
+							Optional:     true,
+							ValidateFunc: validation.StringIsNotEmpty,
```

Comment:
Can we further check whether this `application_id` is an Azure application id (i.e. an UUID) or just an arbitrary string is good enough? It's worth mentioning that even the acctest (API) accepts this value, it doesn't mean the full scenario actually works.

Reply:
Service team confirmed that it will be deprecated in next version. So I removed it.